troubleshooting/troubleshooting-vse-path-too-long-exception.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Path Too Long Exception When Building Project in Microsoft Visual Studio
+description: AppBuilder projects with fully qualified file names longer that 260 symbols fail to build in the extension for Visual Studio.
+type: troubleshooting
+page_title: VS Extension Build Fails When File Path Length Is Longer Than 260 Characters 
```

Comment:
It is better to say that the cloud build fails instead of stating that the issue occurs only in VS. Like this:
"Cloud Build Fails When File Path Length Is Longer Than 260 Characters"